cloud/aws/modules/pgadmin/secrets.tf

```diff
@@ -0,0 +1,43 @@
+# Create a random generated password to use for pgadmin default email.
+resource "random_password" "pgadmin_username" {
+  length  = 10
+  special = false
+  keepers = {
+    version = 1
+  }
+}
+resource "aws_secretsmanager_secret" "pgadmin_username_secret" {
+  name                    = "${var.app_prefix}-civiform_pgadmin_default_username"
+  kms_key_id              = var.secrets_kms_key_arn
+  recovery_window_in_days = var.secrets_recovery_window_in_days
+  tags = {
+    Name = "${var.app_prefix} Civiform pgAdmin Username"
+    Type = "Civiform pgAdmin Username"
+  }
+}
+resource "aws_secretsmanager_secret_version" "pgadmin_username_secret_version" {
+  secret_id     = aws_secretsmanager_secret.pgadmin_username_secret.id
+  secret_string = "${random_password.pgadmin_username.result}@default.login"
+}
+
+# Create a random generated password to use for pgadmin password.
+resource "random_password" "pgadmin_password" {
+  length  = 40
+  special = false
+  keepers = {
+    version = 1
```

Comment:
What does keepers { version = 1 } do?

Reply:
It is an arbitrary map of values that forces recreation when changed: https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password#keepers

I copied these secrets from the existing pattern: https://github.com/civiform/cloud-deploy-infra/blob/main/cloud/aws/templates/aws_oidc/secrets.tf#L27

Reply:
Is it possible/would it makes sense to re-generate new password on each run? That way if password is leaked like admin shares it accidentally - it won't be useful next time pgadmin is deployed.

Reply:
The current behavior is that if the pgadmin secrets have been deployed then destroyed, the next time they get deployed a new random password will be generated

Reply:
Ah. Perfect!